src/priyanka_Panat/assignment14/ShopScanner.java

```diff
@@ -0,0 +1,107 @@
+package priyanka_Panat.assignment14;
+
+import java.util.Scanner;
+
+public class ShopScanner {
+	int totalMaggieQty = 50, totalDosaQty = 43, totalOilQty = 39, totalPanipuriQty = 43,
+			totalMasalaQty = 73;
+
+
+	void requestedQuantity() {
+		Scanner sc = new Scanner(System.in);
```

Comment:
We should not ask, user to give inputs after calling methods, 
We should ask user to give input while calling methods, and we take input once only from main method not in every method